File: app/src/main/java/com/mytlogos/enterprise/viewmodel/ReadEpisodeViewModel.java
package com.mytlogos.enterprise.viewmodel;

import android.app.Application;

import androidx.lifecycle.LiveData;
import androidx.paging.PagedList;

import com.mytlogos.enterprise.model.ReadEpisode;

public class ReadEpisodeViewModel extends RepoViewModel {
    public ReadEpisodeViewModel(Application application) {
        super(application);
    }

    public LiveData<PagedList<ReadEpisode>> getReadEpisodes() {
        return repository.getReadTodayEpisodes();
    }

}

File: app/src/main/java/com/mytlogos/enterprise/viewmodel/ReadEpisodeViewModel.kt
package com.mytlogos.enterprise.viewmodel

import android.app.Application
import androidx.paging.PagingData
import com.mytlogos.enterprise.model.ReadEpisode
import kotlinx.coroutines.flow.Flow

class ReadEpisodeViewModel(application: Application) : RepoViewModel(application) {
    fun getReadEpisodes(): Flow<PagingData<ReadEpisode>> {
        return repository.readTodayEpisodes
    }
}